sanic/app.py

```diff
@@ -411,7 +412,11 @@ def enable_websocket(self, enable=True):
 
         self.websocket_enabled = enable
 
-    def blueprint(self, blueprint, **options):
+    def blueprint(
+        self,
+        blueprint: Union[Blueprint, List[Blueprint], Tuple[Blueprint], BlueprintGroup],
```

Comment:
Maybe i have to specify it separately as a variable?

Reply:
Run `make pretty`

Reply:
Done ! :D